test/views/EditSetViewSpec.scala

```diff
@@ -49,12 +53,17 @@ class EditSetViewSpec extends BaseSpec {
         editSet.entries.size,
         1
       )
+
+      when(mockGovukTable.apply(Table(anyList))).thenReturn(getEditSetTable())
```

Comment:
I had expected that we'd render the whole view, with data, and preferable using the actual messages.

This is an odd hybrid of real and fake.

I wonder if we could be more realistic, and yet still have a quick test?

Perhaps I'm unclear of the general intent here.

Reply:
Removed now as mock objects with govuk component.

Reply:
Confirmed.